src/components/Drawer.js

```diff
@@ -67,7 +67,12 @@ const Drawer = React.createClass({
     }
   },
 
-  _handleCloseClick () {
+  /**
+   * Animate the Drawer closed and then call the onClose callback.
+   *
+   * @returns {Promise} that is resolved when the animation finishes
+   */
+  close () {
```

Comment:
I would rename it back to `_handleCloseClick`. That is our convention to name functions inside components.